cfme/tests/automate/custom_button/test_rest_custom_button.py

```diff
@@ -159,3 +180,40 @@ def test_edit(self, buttons_groups, appliance, from_detail):
             )
             condition.reload()
             assert condition.description == edited[index].description == record[0].description
+
+
+@pytest.mark.meta(blockers=[BZ(1745198)], automates=[1737449])
+def test_associate_unassigned_buttons_rest(appliance, group, buttons):
+    """Test associate unassigned button with group
+
+    Bugzilla:
+        1737449
+        1745198
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/4h
+        caseimportance: medium
+        startsin: 5.10
+        casecomponent: CustomButton
+        tags: custom_button
+    """
+
+    # Add associate unassigned buttons to group with rest
+    set_data = group.set_data
+    set_data["button_order"] = [int(b.id) for b in buttons]
+    data = {"set_data": set_data}
+    group.action.edit(**data)
```

Comment:
Not asserting response?

Reply:
why we need to assert response? actually  test check is with UI buttons associated or not.

Reply:
added :+1: good to have more checks